website/pages/docs/nia/installation/requirements.mdx

```diff
@@ -10,6 +10,8 @@ description: >-
 
 Consul-Terraform-Sync requires a Terraform Provider, a Terraform Module and a running Consul Cluster outside of the consul-terraform-sync daemon. Practitioners can add support for their network infrastructure through Terraform providers. Once network infrastructure support exists, practitioners can add network integrations in the form of Terraform modules.
 
+## Run a Consul Cluster
+
 ## Network Infrastructure (using a Terraform Provider)
```

Comment:
2 headers of the same level (##) right together? Just want to check that was intended.

Reply:
I had moved the "Run a Consul Cluster" from the very bottom of the page to the first item so it doesn't get lost with all of the module content. It's a section to be filled in -- believe Lorna is working on it